backend/curriculum_tracking/management/auto_assign_reviewers.py

```diff
@@ -286,11 +285,13 @@ def get_cards_needing_reviewer_allocation(team):
 def auto_assign_reviewers_based_on_reviewer_team_permission():
     config = NameSpace.get_config(CONFIGURATION_NAMESPACE)
 
-    exclude_users_nested = [
-        Team.objects.get(name=name).users.all()
-        for name in config.EXCLUDE_REVIEWER_PERMISSIONED_USERS_IN_TEAMS
-    ]
-    exclude_users = [user for subset in exclude_users_nested for user in subset]
+    exclude_users = []
+
+    for name in config.EXCLUDE_REVIEWER_PERMISSIONED_USERS_IN_TEAMS:
+        try:
+            exclude_users.append(*Team.objects.get(name=name).users.all())
```

Comment:
This won't work. You can only call append with one argument and you are attempting to unpack a whole queryset. This will only work if the team only has one user. 

```
>>> l = []
>>> ll = [1,2,3]
>>> l.append(*ll)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: list.append() takes exactly one argument (3 given)
```

You either need to use `extend` or you need to append one thing at a time. 

It's often useful to just fire up a python shell to make sure things work the way you think they do.